### ISSUE TITLE
No accessibility / ARIA support

### ISSUE THREAD

**Issue by Ronamo** [MohammadYounes/AlertifyJS#237]
I cannot find any documentation on accessibility options for Alertify.js, nor note any mention of ARIA attributes in the source code.

Is accessibility an upcoming or planned feature?

**Comment by MohammadYounes** [comment]
It will be considered.

**Comment by RichardDavies** [comment]
I'd like to second this request. We use this in some of our government web applications and with the [upcoming DOJ accessibility mandate](https://www.ada.gov/resources/2024-03-08-web-rule/), it would be really nice if this component had better accessibility/ARIA support.
